packages/tools/pluggable-widgets-tools/configs/eslint.js.base.json

```diff
@@ -261,7 +261,7 @@
                 "before": false
             }
         ],
-        "sort-imports": "warn",
+        "sort-imports": ["warn", { "allowSeparatedGroups": true }],
```

Comment:
This change is necessary to allow separating local ("./...") imports from package imports

Reply:
Can you show an example where this change is required?

Reply:
https://github.com/mendix/widgets-resources/blob/master/packages/tools/generator-widget/generators/app/templates/pluggable/web/emptyTemplateJs/src/WidgetName.jsx.ejs

Reply:
Ah ok cool. Can you reproduce it for the eslint.ts.base.json if you didnt yet?

Reply:
`eslint.ts.base.json` does not include `"sort-imports"` thus the rule is disabled.